ibis/backends/clickhouse/tests/snapshots/test_select/test_simple_joins/join_keys0-inner_join/out.sql

```diff
@@ -0,0 +1,5 @@
+SELECT
+  batting.*
```

Comment:
`t1`?

Reply:
Fixed.

Reply:
Did you rename the test case? Asking because the snapshot hasn't changed.

Reply:
I assume yes. How can we remove the outdated snapshots?